editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/behaviours/CamelPersistencyBehaviour.java

```diff
@@ -88,15 +88,10 @@ public void saveDiagram(IProgressMonitor monitor) {
 	 * Creates the diagram from the current model
 	 */
 	public boolean loadModel(IEditorInput input) {
-		IFile camelContextFile = null;
-		
 		IFileEditorInput fileEditorInput = editor.asFileEditorInput(input);
 		if (fileEditorInput != null) {
-			camelContextFile = fileEditorInput.getFile();
-			if (camelContextFile.getFullPath().toOSString().indexOf(".FuseRemoteCamelContextData") == -1) {
```

Comment:
Disclaimer: I don't know enough about the history of this code to know whether it's okay to remove this.